internal/cli/command/completion/cmd.go

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2018 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package completion
+
+import (
+	"os"
+
+	"github.com/banzaicloud/banzai-cli/internal/cli"
+	"github.com/spf13/cobra"
+)
+
+// NewCompletionCommand returns a cobra command for `completion` subcommands.
+func NewCompletionCommand(banzaiCli cli.Cli) *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
I wonder if this command should be hidden. AFAIK most cobra based CLIs hide it. Anyone?

Reply:
kubectl doesn't hide it